libsolidity/interface/CompilerStack.h

```diff
@@ -113,14 +113,15 @@ class CompilerStack: boost::noncopyable
 
 	/// Compiles the source units that were previously added and parsed.
 	/// @returns false on error.
-	bool compile(bool _optimize = false, unsigned _runs = 200);
+	bool compile(
+		bool _optimize = false,
+		unsigned _runs = 200,
+		std::map<std::string, h160> const& _libraries = std::map<std::string, h160>{}
+	);
 	/// Parses and compiles the given source code.
 	/// @returns false on error.
 	bool compile(std::string const& _sourceCode, bool _optimize = false);
```

Comment:
Shouldn't this also have the `_runs` parameter? Is there a need two support both APIs (this and the preceding one)?